amida-tech/indaba-client
# Feature/INBA-500 PM User Group Delete Checks

#### What's this PR do?
Show prompts before allowing user group deletes from pm user groups tab and error if group is assigned to a stage.

#### Related JIRA tickets:
[INBA-500](https://jira.amida-tech.com/browse/INBA-500)

#### How should this be manually tested?
1. Create a project with a user group that is assigned to a stage, and one that isn't
1. Make the project active
1. Try to delete the group that is assigned to a stage. Check that it doesn't work and an error toast appears.
1. Delete the group that is not assigned to a stage. Check that it works after a prompt.
1. Make the project inactive
1. Delete the group that is assigned to a stage. Check that it lets you after showing a prompt.

#### Any background context you want to provide?
#### Screenshots (if appropriate):
